### DISCUSSION TITLE
Playback jump at 10 seconds

### DISCUSSION THREAD

**Discussion by AVTPJ** [videojs/video.js#8197] [Q&A] [answered]
I posted this as an issue on the `http-streaming` repo but haven't heard anything so I thought maybe there's another way to tackle this..

Basically, when streaming HLS from TwentyThree where the video playback jumps forward a fraction of a second, precisely 10 seconds into the video. Here's the link to the issue: https://github.com/videojs/http-streaming/issues/1369

Could someone who has a better understanding of HLS explain what is happening here? Is this something to do with preloading?

Any way around this?

**Comment by amtins** [comment, marked as answer]
@AVTPJ please see my answer https://github.com/videojs/http-streaming/issues/1369#issuecomment-1475965185

**Reply by AVTPJ** [reply]
Thanks for your reply, appreciate it
